api/mod/src/settings.py

```diff
@@ -34,3 +34,7 @@ class Config:
 
 
 SETTINGS = Settings()
+if SETTINGS.database_url.startswith("postgres://"):
+    SETTINGS.database_url = SETTINGS.database_url.replace(
```

Comment:
Why?

Reply:
Heroku automatically sets a DATBASE_URL config variable that holds the connection string to the database. Sadly, it is not editable. But sqlalchemy will only recognise the database type when the string starts with `postgresql`, but Heroku writes only "postgres" so this needs to be replaced

Reply:
okidoki